lib/angular-evaporate.js

```diff
@@ -113,6 +115,16 @@
             this.contentType = this.file.type || 'binary/octet-stream';
             this.$url = ae.$url + this.name;
 
+            function speedStr (size) {
```

Comment:
i've thought about this function again. let's move it to the `AngularEvaporate.prototype` (next to the `$namer` definition). and for an upload just calculate a `$speed` value, which should be a string, like you did initially. if someone wants to replace the function, he'd create his own `ae.$speedStr = function (speed) {...}`. in that concept we'll avoid duplication of functions and data with every upload instance.